client/src/components/MultiJurisdictionAudit/index.test.tsx

```diff
@@ -127,6 +133,185 @@ describe('AA setup flow', () => {
+      })
+      const jurisdisctionInput = screen.getByLabelText('Select a CSV...')
+      const jurisdictionButton = screen.getByRole('button', {
+        name: 'Upload File',
+      })
+      userEvent.click(jurisdictionButton)
+      await screen.findByText('You must upload a file')
+
+      userEvent.upload(jurisdisctionInput, jurisdictionFile)
+      await waitFor(() =>
+        expect(
+          screen.queryByText('You must upload a file')
+        ).not.toBeInTheDocument()
+      )
+
+      userEvent.click(jurisdictionButton)
+      await screen.findByText(/Upload successfully completed/)
+    })
+  })
+
+  it('jurisdisction file upload with error', async () => {
+    const expectedCalls = [
+      aaApiCalls.getUser,
+      ...loadEach,
+      ...loadEach,
+      aaApiCalls.getSettings(auditSettings.blank),
+      aaApiCalls.getJurisdictionFileWithResponse(jurisdictionFileMocks.empty),
+      aaApiCalls.putJurisdictionErrorFile,
+      aaApiCalls.getJurisdictionFileWithResponse(jurisdictionFileMocks.errored),
+    ]
+    await withMockFetch(expectedCalls, async () => {
+      const { container, queryAllByText } = render(
+        <AuthDataProvider>
+          <Router>
+            <AuditAdminViewWithAuth />
+          </Router>
+        </AuthDataProvider>
+      )
+
+      await waitFor(() => {
+        expect(queryAllByText('Participants').length).toBe(2)
+      })
+      const jurisdisctionInput = screen.getByLabelText('Select a CSV...')
+      const jurisdictionButton = screen.getByRole('button', {
+        name: 'Upload File',
+      })
+      userEvent.click(jurisdictionButton)
+      await screen.findByText('You must upload a file')
+
+      userEvent.upload(jurisdisctionInput, jurisdictionErrorFile)
+      await waitFor(() =>
+        expect(
+          screen.queryByText('You must upload a file')
+        ).not.toBeInTheDocument()
+      )
+
+      userEvent.click(jurisdictionButton)
+      await screen.findByText('Invalid CSV')
+    })
+  })
+
+  it('standardized contests file upload success', async () => {
+    const expectedCalls = [
+      aaApiCalls.getUser,
+      ...loadEach,
+      ...loadEach,
+      aaApiCalls.getSettings(auditSettings.blankBallotComparison),
+      aaApiCalls.getJurisdictionFile,
+      aaApiCalls.getStandardizedContestsFileWithResponse(
+        standardizedContestsFileMocks.empty
+      ),
+      aaApiCalls.putStandardizedContestsFile,
+      aaApiCalls.getStandardizedContestsFileWithResponse(
+        standardizedContestsFileMocks.processed
+      ),
+    ]
+    await withMockFetch(expectedCalls, async () => {
+      const { container, queryByText, queryAllByText } = render(
+        <AuthDataProvider>
+          <Router>
+            <AuditAdminViewWithAuth />
+          </Router>
+        </AuthDataProvider>
+      )
+
+      // check file upload of jurisdiction
+      await waitFor(() => {
+        expect(queryAllByText(/Upload successfully completed/).length).toBe(1)
+      })
```

Comment:
simpler to use `await screen.findByText(...)` like you did on L311. i think there are a few other places where you use this waitfor/expect/query pattern that could be simplified